### DISCUSSION TITLE
Shared public list is asking for login

### DISCUSSION THREAD

**Discussion by don-ferris** [Kovah/LinkAce#807] [General]
I’m a new user and I’m really LOVING LinkAce. Thank you, @Kovah!! I struggled way more than I should have to get it setup but now that it’s working, it’s everything I wanted! 

I have a list that I created for Wish List items. I sent it to my wife but when she clicks the link, it’s asking her to login. “Is Private” is set to “No”…
![IMG_1221](https://github.com/Kovah/LinkAce/assets/109878120/4458d628-e40f-4bb1-81c8-798a77773820)

In general, my preference is for all my lists/bookmarks to be public unless I specifically set them to private. Ergo…
![IMG_1222](https://github.com/Kovah/LinkAce/assets/109878120/5b4aa3b2-2d70-4e21-bff8-d2c5df0e0d22)
What am I doing wrong?

**Comment by Kovah** [comment]
Great to hear that you like LinkAce! Did you enable the guest mode in the system settings already?

If a list is public you should be able to access it like here: https://demo.linkace.org/guest/lists/2

**Reply by Kovah** [reply]
Ping @don-ferris
